src/asciidoc/docs/develop/publishLocal.adoc

```diff
@@ -0,0 +1,240 @@
+= Publishing a project you developed locally
+:page-slug: publish-locally-developed
+:page-description: How to push local changes to the origin repository and pull them into the Koji editor, so you can publish the project.
+
+In the <<work-locally#, previous section>> of this tutorial, you learned how to use Git to clone a Koji project to your development machine and how to install and launch a Node.js server to run your project locally.
+
+In this section, you will
+// tag::description[]
+learn how to push the changes that you make locally to your project's origin repository, and how to pull the latest version of your project into the Koji editor, where you can publish it to the world.
+// end::description[]
+
+NOTE: It is recommended that you test the publishing process before you get deep into development locally.
+You can make a minor change, push it back to the Koji servers, and check that the change is visible in the published app on Koji.
+
+== Learning objectives
+
+By the end of the section, you should feel comfortable:
+
+* Making and testing changes locally.
+* Publishing your changes to the Koji server.
+* Pushing your changes to the origin repository, and then pulling them into the Koji editor.
+
+== Prerequisites
+
+* Follow the first two sections to set up the https://withkoji.com/~Svarog1389/rxkd[Falling Objects] game on your development machine.
+
+== Making a change in the source code
+
+Let's make a small change in the https://withkoji.com/~Svarog1389/rxkd[Falling Objects] game, just for testing purposes.
+
+. Open `frontend\src\Components\View\LeaderboardMini.js`.
+Change  `<Title>{"Top Players"}</Title>` to `<Title>{"Leaderboard"}</Title>` and save the file.
+
+. In your browser, go to `\http://0.0.0.0:8080`, and make sure the leaderboard shows the new title.
+
+== Publishing your changes to the Koji server
```

Comment:
Since you no longer need to publish from the editor, the flow of the sub-sections in “Publishing your changes to the Koji server” doesn’t quite work. Specially, I would make Testing an h2 and add an h2 that unites the two Koji code editor topics (pulling into editor / updating Node modules). I'll make some inline comments too.